controllers/virtualmachinegrouppublishrequest/virtualmachinegrouppublishrequest_controller.go

```diff
@@ -95,47 +108,48 @@ func NewReconciler(
 func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (_ ctrl.Result, reterr error) {
 	ctx = pkgcfg.JoinContext(ctx, r.Context)
 
-	obj := &vmopv1.VirtualMachineGroupPublishRequest{}
-	if err := r.apiReader.Get(ctx, req.NamespacedName, obj); err != nil {
+	vmGroupPublishRequest := &vmopv1.VirtualMachineGroupPublishRequest{}
+	if err := r.apiReader.Get(ctx, req.NamespacedName, vmGroupPublishRequest); err != nil {
```

Comment:
Do you know why this is using the API reader client? The publish request controller needed that due to some funky VC API stuff but not clear to me if that applies to this object.

Reply:
no and I dont see a reason to use api reader directly. updated.